Decred Wallet/Features/Receive/ReceiveViewController.swift

```diff
@@ -0,0 +1,171 @@
+//  ReceiveViewController.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import Foundation
+import UIKit
+import Dcrlibwallet
+
+class ReceiveViewController: UIViewController, UIDocumentInteractionControllerDelegate {
```

Comment:
I meant to ask, what's `UIDocumentInteractionControllerDelegate` for? I believe it can be removed?

Reply:
I've removed it